src/OrbitSshQt/SftpCopyToLocalOperation.cpp

```diff
@@ -72,34 +117,15 @@ outcome::result<void> SftpCopyToLocalOperation::startup() {
       ABSL_FALLTHROUGH_INTENDED;
     }
     case State::kLocalFileOpened: {
-      constexpr size_t kReadBufferMaxSize = 1 * 1024 * 1024;
-
-      while (true) {
-        OUTCOME_TRY(auto&& read_buffer, sftp_file_->Read(kReadBufferMaxSize));
-        if (read_buffer.empty()) {
-          // This is end of file
-          SetState(State::kLocalFileWritten);
-          break;
-        }
-
-        local_file_.write(read_buffer.data(), read_buffer.size());
-      }
-      ABSL_FALLTHROUGH_INTENDED;
-    }
-    case State::kLocalFileWritten: {
-      local_file_.close();
-      SetState(State::kLocalFileClosed);
-      ABSL_FALLTHROUGH_INTENDED;
-    }
-    case State::kLocalFileClosed: {
-      OUTCOME_TRY(sftp_file_->Close());
-      about_to_shutdown_connection_ = std::nullopt;
-      SetState(State::kDone);
-      ABSL_FALLTHROUGH_INTENDED;
+      SetState(State::kStarted);
+      break;
     }
+    case State::kStarted:
```

Comment:
Is there a reason to distinguish between kLocalFileOpened and kStarted? It seems they are the same state, right?

Reply:
Yes, the state `kLocalFileOpened` did not do anything. I removed it and renamed some other states to be more readable.